Hazel/premake5.lua

```diff
@@ -43,12 +43,32 @@ project "Hazel"
 		"GLFW",
 		"Glad",
 		"ImGui",
-		"opengl32.lib"
 	}
 
+	filter "system:linux"
+		-- pic "on"
+
+		links 
+		{ 
+			"Xrandr",
+			"Xi",
+			"GLU",
+			"GL",
+			"X11"
+		}
+
+		defines
```

Comment:
`HZ_PLATFORM_LINUX` should be defined for the Hazel library since this is a Premake oriented project.

Reply:
Same comment as above